patches/chrome-browser-ui-views-tabs-alert_indicator.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/tabs/alert_indicator.h b/chrome/browser/ui/views/tabs/alert_indicator.h
+index 6327526fff9ffa05959df543a2441ffc6df83134..8d394af1744f80d19ecf29b448b1a154a3544885 100644
+--- a/chrome/browser/ui/views/tabs/alert_indicator.h
++++ b/chrome/browser/ui/views/tabs/alert_indicator.h
+@@ -44,6 +44,7 @@ class AlertIndicator : public views::ImageView {
+   View* GetTooltipHandlerForPoint(const gfx::Point& point) override;
+ 
+  private:
++  friend class BraveAlertIndicator;
```

Comment:
I'm not seeing where we're calling private methods on `AlertIndicator`. Am I missing something?

Reply:
I see, for `parent_tab_`

Reply:
yup, that's it.